blog.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adoptadog Home</title>
+  </head>
+  <body>
+    <header>
+      <img src="images/adoptadog-logo.svg" alt="">
+      <a href="index.html"><h1>Adoptadog</h1></a>
+      <h2>Seattle's premiere dog adoption agency</h2>
+      <aside>
+        <!--icon for checkout-->
+        <!--total, ecommerce feature-->
+        <a class="#" href="checkout.html">checkout</a>
+      </aside>
+    </header>
+    <nav>
+      <a class="#" href="index.html"><h6>Home</h6></a>
+      <a class="#" href="gallery.html"><h6>Dogs</h6></a>
+      <a class="#" href="blog.html"><h6>Blog</h6></a>
+    </nav>
+    <article class="#">
+      <h3>Adoptadog Blog</h3>
+      <div class="#">
```

Comment:
You'll want to use real class names. I'm sure you're aware of it, but have to comment, just in case.